### ISSUE TITLE
Node 12 doesn't support ES2020 Conditional Chaining

### ISSUE THREAD

**Issue by thomaspirard** [colinbendell/homebridge-blink-for-home#56]
Hello guys,
My plugin stopped working since yesterday.
This is my log 
![grabilla g14924](https://user-images.githubusercontent.com/79282148/108720233-d94c6d80-7520-11eb-8ac0-c06e6c799d52.png)
Thnks for helping

**Comment by colinbendell** [comment]
Can you tell me what version of homebridge you are running?

**Comment by thomaspirard** [comment]
Hello,
I use 1.3.0 version

**Comment by kevanwm** [comment]
Also seeing this issue in `homebridge-blink-for-home v3.7.1` & `Homebridge 1.3.0` (I was seeing it in `Homebridge v1.2.5`, as well)


```
[2/22/2021, 7:41:22 PM] ====================
[2/22/2021, 7:41:22 PM] ERROR LOADING PLUGIN homebridge-blink-for-home:
[2/22/2021, 7:41:22 PM] /usr/lib/node_modules/homebridge-blink-for-home/src/inifile.js:10
            const [name, ...value] = line?.trim()?.split(/=/);
                                          ^

SyntaxError: Unexpected token '.'
    at wrapSafe (internal/modules/cjs/loader.js:915:16)
    at Module._compile (internal/modules/cjs/loader.js:963:27)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1027:10)
    at Module.load (internal/modules/cjs/loader.js:863:32)
    at Function.Module._load (internal/modules/cjs/loader.js:708:14)
    at Module.require (internal/modules/cjs/loader.js:887:19)
    at require (internal/modules/cjs/helpers.js:74:18)
    at Object.<anonymous> (/usr/lib/node_modules/homebridge-blink-for-home/src/blink-api.js:6:17)
    at Module._compile (internal/modules/cjs/loader.js:999:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1027:10)
[2/22/2021, 7:41:22 PM] ====================
```

**Comment by thomaspirard** [comment]
> Also seeing this issue in `homebridge-blink-for-home v3.7.1` & `Homebridge 1.3.0` (I was seeing it in `Homebridge v1.2.5`, as well)
> 
> ```
> [2/22/2021, 7:41:22 PM] ====================
> [2/22/2021, 7:41:22 PM] ERROR LOADING PLUGIN homebridge-blink-for-home:
> [2/22/2021, 7:41:22 PM] /usr/lib/node_modules/homebridge-blink-for-home/src/inifile.js:10
>             const [name, ...value] = line?.trim()?.split(/=/);
>                                           ^
> 
> SyntaxError: Unexpected token '.'
>     at wrapSafe (internal/modules/cjs/loader.js:915:16)
>     at Module._compile (internal/modules/cjs/loader.js:963:27)
>     at Object.Module._extensions..js (internal/modules/cjs/loader.js:1027:10)
>     at Module.load (internal/modules/cjs/loader.js:863:32)
>     at Function.Module._load (internal/modules/cjs/loader.js:708:14)
>     at Module.require (internal/modules/cjs/loader.js:887:19)
>     at require (internal/modules/cjs/helpers.js:74:18)
>     at Object.<anonymous> (/usr/lib/node_modules/homebridge-blink-for-home/src/blink-api.js:6:17)
>     at Module._compile (internal/modules/cjs/loader.js:999:30)
>     at Object.Module._extensions..js (internal/modules/cjs/loader.js:1027:10)
> [2/22/2021, 7:41:22 PM] ====================
> ```

I'm sad because the plugin worked really good for 3 weeks (I have homebridge for 4 weeks)

**Comment by colinbendell** [comment]
Ugh. Yea, that's ES2020 syntax that's only supported on Node 14+. I'll revert to Node 12 syntax support.

**Comment by colinbendell** [comment]
I have reverted to Node 12 LTS compatible syntax (ES2019). 

Shipped in v3.7.2
